### ISSUE TITLE
Copying one packet at a time is too slow. We want to copy the packets in the cache when the cache is full?

### ISSUE THREAD

**Issue by MaoPann** [ntop/PF_RING#638]

**Comment by laoyang103** [comment]
@MaoPann which layer is your cache in? how much  your packet speed? in my test zc can run 6000000packet/s on multi application

**Comment by MaoPann** [comment]
L7

**Comment by cardigliano** [comment]
Could you rephrase? What do you men with cache?

**Comment by laoyang103** [comment]
@cardigliano perhaps he means block copy(multi packet) from ZC space(kernel) to user application

**Comment by MaoPann** [comment]
yes

**Comment by MaoPann** [comment]
Copying one packet at a time is too slow.Will greatly reduce our processing speed, if each packet is copied once。

**Comment by cardigliano** [comment]
ZC includes API to work in burst (pfring_zc_recv_pkt_burst) however you still have to handle packets individually

**Comment by MaoPann** [comment]
Can you tell me the address range of the independent buffer when storing packets

**Comment by MaoPann** [comment]
Maybe I can copy the entire stand-alone cache buffer

**Comment by cardigliano** [comment]
What do you mean with address range exactly? Please note buffers are not contiguous, thus we cannot provide a range.
